pkg/utils/o11y/controller.go

```diff
@@ -326,7 +395,7 @@ func (h *SuiteController) QuayImagePushDeployment(quayOrg, secret, namespace str
 								{Name: "STORAGE_DRIVER", Value: "vfs"},
 							},
 							Command: []string{"/bin/sh", "-c"},
-							Args:    []string{h.getImagePushScript(secret, quayOrg)},
+							Args:    []string{h.getImagePushScript(secret, quayOrg) + "; while true; do sleep 30; done"},
```

Comment:
Why is this necessary if the script already sleeps for 1m?

Reply:
Sometimes there is container restart post completion making the pod inaccessible by QueryThanosAPI function. this will keep the pod alive till the end of the test.